src/kate_with_ai/kate_with_ai.h

```diff
@@ -40,7 +40,7 @@ class kate_with_ai_view : public QWidget
 public:
   explicit kate_with_ai_view(kate_with_ai * plugin, KTextEditor::MainWindow * main_window);
 
-private slots:
+private Q_SLOTS:
```

Comment:
I doubt this change is necessary.
how is that possible qt6 is requiring very old qt macros, qt preprocessor in qt5 was handling already slots, emit etc ...